spacy_llm/tasks/__init__.py

```diff
@@ -7,6 +10,35 @@
 from .summarization import SummarizationTask, make_summarization_task
 from .textcat import TextCatTask, make_textcat_task
 
+_LATEST_TASKS = (
+    "spacy.NER.v2",
+    "spacy.NoOp.v1",
```

Comment:
Do we need a registered `llm_NoOp` factory?

Reply:
Hm, I don't think so. Removed in https://github.com/explosion/spacy-llm/pull/243/commits/899966d7aa571a1a56711db83a2485e509cf1230.